app/layer_example/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+set(ARM_DIR "${CMAKE_SOURCE_DIR}/3rdparty/ComputeLibrary")
+
+add_executable(ConvolutionLayer ConvolutionLayer.cpp)
+add_executable(ElementwiseLayer ElementwiseLayer.cpp)
+
+include_directories(${ARM_DIR})
+include_directories(${ARM_DIR/include})

```

The root cause of "half" inclusion issue is this line. It should be like this:
```suggestion
include_directories(${ARM_DIR}/include)
```
Please fix in here and in other PRs